src/linagora.esn.contact/app/services/live-update/contact-live-update-initializer.service.spec.js

```diff
@@ -26,9 +26,7 @@ describe('The ContactLiveUpdateInitializer service', function() {
           }
         ]
       },
-      ready: {
-        then: function() {}
-      }
+      ready: Promise.resolve({})
```

Comment:
$q.when({}) should work here. Tbh I think we should not mix Promise and $q in AngularJS. We should just only use $q.